packages/manfred-common/src/doc-parts/studies-section/studies-section.helpers.ts

```diff
@@ -0,0 +1,5 @@
+import { sortByDate } from '../../helpers';
```

Comment:
Use '@/helpers"

Reply:
Same as with experience. With import in this way I get test failures